docs/jwst/undersampling_correction/description.rst

```diff
@@ -49,3 +46,4 @@ Output product
 --------------
 The output is a new copy of the input `~jwst.datamodels.RampModel`, with the corrections applied
```

Comment:
Hmmm, not sure I would characterize what this step does as "corrections". It's really just adding DQ flags, so that other steps later on do the right thing. So perhaps rephrase this to "with the updated DQ flags added to the GROUPDQ array."